x-pack/solutions/security/plugins/elastic_assistant/server/lib/prompt/saved_object_mappings.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectsType } from '@kbn/core/server';
+import { promptSavedObjectType } from '../../../common/constants';
+
+export const promptSavedObjectMappings: SavedObjectsType['mappings'] = {
```

Comment:
Hi @andrew-goldstein. @patrykkopycinski suggested we use saved objects for the prompts. Here is my proposal for the mappings. Please let me know what you think